roofit/roofitcore/src/FitHelpers.cxx

```diff
@@ -35,6 +35,7 @@
 #include <RooMinimizer.h>
 #include <RooRealVar.h>
 #include <RooSimultaneous.h>
+#include <RooFormulaVar.h>
```

Comment:
A general note: as a new source file in RooFit, the `FitHelpers.cxx` file is formatted with clang-format using the ROOT style:
https://github.com/root-project/root/blob/master/.clang-format

Can you please re-format it after making the changes?

Note that doesn't apply to the `RooDerivative.cxx` source file, because that one is old and reformatting it would mess up the history (which is useful to keep for `git blame`).

Reply:
Done.